lms/djangoapps/grades/migrations/0001_initial.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import coursewarehistoryextended.fields
+import django.utils.timezone
+import model_utils.fields
+import xmodule_django.models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='PersistentSubsectionGrade',
+            fields=[
+                ('created', model_utils.fields.AutoCreatedField(default=django.utils.timezone.now, verbose_name='created', editable=False)),
+                ('modified', model_utils.fields.AutoLastModifiedField(default=django.utils.timezone.now, verbose_name='modified', editable=False)),
+                ('id', coursewarehistoryextended.fields.UnsignedBigIntAutoField(serialize=False, primary_key=True)),
+                ('user_id', models.IntegerField()),
+                ('course_id', xmodule_django.models.CourseKeyField(max_length=255)),
```

Comment:
We're inconsistent throughout the codebase about the terms "key" and "id." Here we have a `usage_key` and a `course_id` even though they are both `___KeyField`s.